js_modules/dagit/packages/core/src/assets/__stories__/OverdueTag.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import {MockedProvider, MockedResponse} from '@apollo/client/testing';
+import {Box} from '@dagster-io/ui';
+import React from 'react';
+
+import {createAppCache} from '../../app/AppCache';
+import {
+  LiveDataForNodeMaterialized,
+  LiveDataForNodeMaterializedAndOverdue,
+  LiveDataForNodeNeverMaterialized,
+  LiveDataForNodeMaterializedAndFresh,
+} from '../../asset-graph/__fixtures__/AssetNode.fixtures';
+import {buildAssetKey, buildAssetNode, buildFreshnessPolicy, buildQuery} from '../../graphql/types';
+import {OVERDUE_POPOVER_QUERY, OverdueTag} from '../OverdueTag';
+import {OverduePopoverQuery, OverduePopoverQueryVariables} from '../types/OverdueTag.types';
+
+// eslint-disable-next-line import/no-default-export
+export default {component: OverdueTag};
+
+const mockFreshnessPolicy = buildFreshnessPolicy({
+  cronSchedule: '30 2 * * *',
+  cronScheduleTimezone: null,
+  maximumLagMinutes: 2,
+});
+
+function buildOverduePopoverMock(): MockedResponse<
+  OverduePopoverQuery,
+  OverduePopoverQueryVariables
+> {
+  return {
+    request: {
+      query: OVERDUE_POPOVER_QUERY,
+      variables: {
+        assetKey: {path: ['inp8']},
+        timestamp: LiveDataForNodeMaterializedAndOverdue.lastMaterialization!.timestamp,
+      },
+    },
+    result: {
+      data: buildQuery({
+        assetNodeOrError: buildAssetNode({
+          id: 'test.py.repo.["inp8"]',
+          freshnessPolicy: mockFreshnessPolicy,
+          assetMaterializationUsedData: [
+            {
+              timestamp: '1684596878856',
+              assetKey: buildAssetKey({
+                path: ['inp1'],
+              }),
+              downstreamAssetKey: buildAssetKey({
+                path: ['inp8'],
+              }),
+              __typename: 'MaterializationUpstreamDataVersion',
+            },
+            {
+              timestamp: '1686943124049',
+              assetKey: buildAssetKey({
+                path: ['inp2'],
+              }),
+              downstreamAssetKey: buildAssetKey({
+                path: ['inp8'],
+              }),
+              __typename: 'MaterializationUpstreamDataVersion',
+            },
+          ],
+          __typename: 'AssetNode',
```

Comment:
I think you remove the __typename here since you're using the builder

Reply:
Nice! Good call